src/coreclr/debug/daccess/request.cpp

```diff
@@ -4727,11 +4732,15 @@ HRESULT ClrDataAccess::GetReJITInformation(CLRDATA_ADDRESS methodDesc, int rejit
     SOSDacLeave();
 
     return hr;
+#else
+    return E_NOTIMPL;
+#endif // FEATURE_CODE_VERSIONING
 }
 
 
 HRESULT ClrDataAccess::GetProfilerModifiedILInformation(CLRDATA_ADDRESS methodDesc, struct DacpProfilerILData *pILData)
 {
+#ifdef FEATURE_CODE_VERSIONING
```

Comment:
This ifdef can be smaller (just `CodeVersionManager* pCodeVersionManager = pMD->GetCodeVersionManager();` block) so that the method succeeds as if there was no code versioning event.